public/app/features/alerting/unified/rule-list/GrafanaRuleLoader.tsx

```diff
@@ -0,0 +1,64 @@
+import { GrafanaPromRuleDTO, PromRuleType } from 'app/types/unified-alerting-dto';
+
+import { GrafanaRulesSource } from '../utils/datasource';
+
+import { AlertRuleListItem, RecordingRuleListItem, UnknownRuleListItem } from './components/AlertRuleListItem';
+
+interface GrafanaRuleLoaderProps {
+  rule: GrafanaPromRuleDTO;
+  groupName: string;
+  // TODO: How to improve this?
```

Comment:
(What kind of improvements were you thinking of here btw?)

Reply:
So, we already pass `groupIdentifier` which contains a `namespace` property, but for GMA namespace only contains a folder's `uid` whereas for DS rules it contains name (because we don't have anything else) 

Maybe it's okay as it is. I actually don't have any better ideas which wouldn't make the code more complicated